src/platform/packages/private/kbn-esql-editor/src/esql_editor.tsx

```diff
@@ -585,6 +590,20 @@ export const ESQLEditor = memo(function ESQLEditor({
     }
   }, [isLoading, isQueryLoading, parseMessages, code]);
 
+  useEffect(() => {
+    async function fetchLicense() {
+      try {
+        const ls = await kibana.services?.esql?.getLicense();
+        if (!isEqual(license, ls)) {
+          setLicense(ls);
+        }
+      } catch (error) {
+        // failed to fetch
```

Comment:
Silent catch?

Reply:
yes I dont want to do anything if it fails